client/src/index.css

```diff
@@ -20,7 +20,12 @@ body {
   font-family: "Poppins", sans-serif;
   font-weight: 800;
   box-sizing: border-box;
-  padding: 2rem;
+  /* Padding was interfering with background for Banner component. Is this necessary here? */
```

Comment:
it's not necessary here, it's just all the pages but the one component will have padding around it, 
you could override it in the banner component.

like this 
```jsx
    <PageWrapper>
      <BannerLogo src={logo} />
      <BackgroundWrapper>
        <TitleText>{title}</TitleText>
      </BackgroundWrapper>
    </PageWrapper>
```
also in `if` block;

and styled
```
export const PageWrapper = styled.div`
  margin: -2rem;
`;
```

this will fix it, and return the padding for the body

Reply:
I swear that we tried something like that yesterday, but we just put it on the background wrapper component itself and it works, so thanks!

Reply:
@Oliversw I think I tried it in the `BackgroundWrapper`, it did work, you need to put for the parent div, (don't forget to uncomment `padding` in `client/src/index.css`